source/tests/system/nirfmxwlan_driver_api_tests.cpp

```diff
@@ -1152,6 +1163,54 @@ TEST_F(NiRFmxWLANDriverApiTests, TXPMIMOFromExample_FetchData_DataLooksReasonabl
   EXPECT_LT(0.0, txp_fetch_power_trace_response.power(0));
 }
 
+TEST_F(NiRFmxWLANDriverApiTests, AnalyzeNWaveformsIQ_FetchData_DataLooksReasonable)
+{
+  const auto num_frequency_segments = 1;
+  const auto num_receive_chains = 2;
+  double centerFrequency[2] = {5.18e9, 5.26e9};
+  std::vector<float64> IQx0{0, 0};
```

Comment:
nit: I try not to let `IQ` blow up my naming too much. `iq_x0`, `iq_dx`, `iq_size`, `iq` would work too.

Reply:
updated